plugins/modules/identity/keycloak/keycloak_user_rolemapping.py

```diff
@@ -0,0 +1,400 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, Dušan Marković (@bratwurzt)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: keycloak_user_rolemapping
+
+short_description: Allows administration of Keycloak user_rolemapping with the Keycloak API
+
+version_added: 5.3.0
+
+description:
+    - This module allows you to add, remove or modify Keycloak user_rolemapping with the Keycloak REST API.
+      It requires access to the REST API via OpenID Connect; the user connecting and the client being
+      used must have the requisite access rights. In a default Keycloak installation, admin-cli
+      and an admin user would work, as would a separate client definition with the scope tailored
+      to your needs and a user having the expected roles.
+
+    - The names of module options are snake_cased versions of the camelCase ones found in the
+      Keycloak API and its documentation at U(https://www.keycloak.org/docs-api/8.0/rest-api/index.html).
+
+    - Attributes are multi-valued in the Keycloak API. All attributes are lists of individual values and will
+      be returned that way by this module. You may pass single values for attributes when calling the module,
+      and this will be translated into a list suitable for the API.
+
+    - When updating a user_rolemapping, where possible provide the role ID to the module. This removes a lookup
+      to the API to translate the name into the role ID.
+
+
+options:
+    state:
+        description:
+            - State of the user_rolemapping.
+            - On C(present), the user_rolemapping will be created if it does not yet exist, or updated with the parameters you provide.
+            - On C(absent), the user_rolemapping will be removed if it exists.
+        default: 'present'
+        type: str
+        choices:
+            - present
+            - absent
+
+    realm:
+        type: str
+        description:
+            - They Keycloak realm under which this role_representation resides.
+        default: 'master'
+
+    target_username:
+        type: str
+        description:
+            - Username of the user roles are mapped to.
+            - This parameter is not required (can be replaced by uid for less API call).
+
+    uid:
+        type: str
+        description:
+            - Id of the user to be mapped.
+            - This parameter is not required for updating or deleting the rolemapping but
+              providing it will reduce the number of API calls required.
+
+    service_account_user_client_id:
+        type: str
+        description:
+            - client_id of the service-account-user to be mapped.
```

Comment:
```suggestion
            - Client ID of the service-account-user to be mapped.
```
Or use `I(...)` to refer to an option name.